docs/concepts/html.md

```diff
@@ -9,20 +9,32 @@ The `html!` macro allows you to write HTML and SVG code declaratively. It is sim
 
 **Important notes**
 
-1. The `html!` macro only accepts one root html node \(you can counteract this by
+1. The `html!` macro only accepts a single root HTML node \(this obstacle is easily overcome by
    [using fragments or iterators](html/lists.md)\)
 2. An empty `html! {}` invocation is valid and will not render anything
-3. Literals must always be quoted and wrapped in braces: `html! { "Hello, World" }`
+3. Literals must always be wrapped in quotes as well as braces (i.e.
+`html! { <p>{"Hello, World"}</p> }` is valid, but not `html! { <p>Hello, World</p> }` or
+`html! { <p>"Hello, World"</p> }`).
 
 :::note
-The `html!` macro can reach the default recursion limit of the compiler. If you encounter compilation errors, add an attribute like `#![recursion_limit="1024"]` in the crate root to overcome the problem.
+The requirement to need braces and quotes was not a deliberate design choice (just in case you're
+wondering)! It's needed in order to make parsing the tokens fed into the `html!` macro possible.
```

Comment:
The quotes are definitely required (the alternative is a nightmare to implement), but we could totally get rid of the braces.

Reply:
👍🏼